BHoMAnalytics_UI/CaptureProjectData.xaml.cs

```diff
@@ -43,57 +43,132 @@ namespace BH.UI.Analytics
     /// </summary>
     public partial class CaptureProjectData : Window
     {
-        public CaptureProjectData()
+        #region Constructor
+        public CaptureProjectData(string uiName)
         {
             InitializeComponent();
-            this.ShowDialog();
+            VersionTextBlock.Text = $"BHoM Version: {BH.Engine.Base.Query.BHoMVersion()}";
+            NonProjectListBox.ItemsSource = Enum.GetValues(typeof(NonProjectOption));
+            this._uiName = uiName;
+            
+
+            if (_uiName == "Grasshopper")
+            {
+                GrassHopperUIText.Visibility = Visibility.Visible;
+            }
+            if (_uiName == "Revit")
+            {
+                BHoMUseTextBlock.Text = "Please indicate purpose of this use of BHoM Tools for Revit";
+            }
+
             ProjectBtn.Focus();
+            this.ShowDialog();
         }
+        #endregion
+
+        #region Properties
+        public string Version;
+        public string _uiName;
+
+        public int _initialHeight = 240;
```

Comment:
Do these need to be public?
We should follow convention on them all starting with an _